experiment/simulation/js/js/materials.js

```diff
@@ -0,0 +1,39 @@
+import * as THREE from "https://threejsfundamentals.org/threejs/resources/threejs/r115/build/three.module.js";
```

Comment:
This looks an interesting way to include js. Is it better to load it as part of the HTML and then impor it using the regular syntax?